packages/config/package.json

```diff
@@ -52,12 +53,18 @@
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "merge-stream": "2.0.0",
-    "rollup": "4.3.0",
-    "rollup-plugin-dts": "5.0.0",
-    "rollup-plugin-esbuild": "5.0.0",
-    "rollup-plugin-node-externals": "5.0.2",
+    "rollup": "4.21.2",
+    "rollup-plugin-dts": "6.1.1",
+    "rollup-plugin-esbuild": "6.1.1",
+    "rollup-plugin-node-externals": "7.1.3",
     "sucrase": "3.34.0",
     "tslib": "2.6.2",
     "typescript": "^5.2.2"
+  },
+  "overrides": {
```

Comment:
When I last checked a few months ago, NPM did only respect overrides defined in the root `package.json`. Did that change in a recent NPM version? If so, we still can't rely on it until we require the new version...